tf_seal/cc/kernels/seal_kernels.cc

```diff
@@ -79,6 +79,58 @@ Status LookupOrCreateWrapper(OpKernelContext* ctx,
                                                      });
 }
 
+class SealSavePbOp : public OpKernel {
+ public:
+  explicit SealSavePbOp(OpKernelConstruction* ctx): OpKernel(ctx) {}
+
+
+  void Compute(OpKernelContext* ctx) override {
+      const PublicKeysVariant* key = nullptr;
+    OP_REQUIRES_OK(ctx, GetVariant(ctx, 1, &key));
+      Tensor* out0;
+    OP_REQUIRES_OK(ctx, ctx->allocate_output(0, TensorShape{}, &out0));
+    seal::PublicKey publicKey(key->public_key);
+    std::filebuf fb;
+    fb.open("public_key", std::ios::out);
```

Comment:
we should make the filename an attribute or ideally an input (string scalar)

Reply:
@mortendahl , In this case, If I understood, the seal_ops, I must declared on this way
    REGISTER_OP("SealSavePublickey")
       .Input("nameSavePK: string")
       .Input("pub_key: variant")    
       .Output("out:variant")
       .SetIsStateful();
Then on kernel I must do this?
const Tensor& input_tensor=ctx->input(0);
      std::string input = input_tensor.flat<std::string>();

Or how?. I see on the last line I have a mistake. Thanks for all

Reply:
see eg reply on slack: I think you may have to use a slightly more complex expression, eg like so: https://github.com/tf-encrypted/tf-seal/blob/master/tf_seal/cc/kernels/seal_kernels.cc#L44 (since you are working with tensors, even if flattened)

Reply:
Done. @mortendahl , I have pushed it